### DISCUSSION TITLE
I can't transfert a discussion

### DISCUSSION THREAD

**Discussion by qdequele** [community/community#5729] [Discussions]
I'm trying to transfer a discussion from a public repository [meilisearch/meilisearch](https://github.com/meilisearch/MeiliSearch/discussions) to another public repository [meilisearch/product](https://github.com/meilisearch/product/discussions), but I have the error "_Could not transfer the discussion to meilisearch/product at this time_". Moreover, I've admin rights on both repositories. Is it a temporary Github error, or did I made something wrong?

**Comment by curquiza** [comment]
Hello! The issue is still present and kind of annoying for us! We are available to give you any additional information you would need.

Here is the screenshot of the error:
<img width="964" alt="Capture d’écran 2021-10-20 à 18 05 35" src="https://user-images.githubusercontent.com/20380692/138129965-271309a9-cc25-4626-a01b-1f86753d81f3.png">

**Reply by gmourier** [reply]
I second that, it forces us to have to rewrite a similar discussion on the target repository, we lose all initial comments on the target discussion by doing so and we also have to delete the original discussion or comment on it that we have created a discussion elsewhere.

It seems that it's a permission error but we didn't find the cause after multiple investigations.

Thank you for your help, it would greatly enhance the experience with our contributors. 😇

**Comment by curquiza** [comment]
Just to remind we have still the bug when trying to transfer discussion like this one: https://github.com/meilisearch/MeiliSearch/discussions/1096 from `MeiliSearch` repo to `product` repo.
